popup/popup.css

```diff
@@ -26,6 +26,10 @@ body {
   margin: 0;
 }
 
+.firefox, .firefox body:not(.search-results-shown) {
```

Comment:
What does it do?

Reply:
The popup when the height adjusts to accommodate the menus is ridiculously bad looking in Firefox. IIRC, stonecrusher had mentioned this a while back. Seems like any time any kinda calculation is used, FF is just slower and glitchier at executing. I made a [screen record](https://streamable.com/jsovj), which basically shows it blinking scrollbars in and out as the height adjusts.

 We can get rid of those by hiding overflow with the above CSS, but it's still kinda glitchy at the bottom as the popup expands and contracts. Maybe little bugs like this are just the status quo for FF users, and it's fine. I don't see any way around it anyway, but maybe someone else does.